examples/sound/Sound.cpp

```diff
@@ -87,8 +88,11 @@ int main()
     // Play a sound
     playSound();
 
-    // Play a music
-    playMusic();
+    // Play music from an ogg file
+    playMusic("orchestral.ogg");
+    
+    // Play music from an flac file
+    playMusic("song.flac");
```

Comment:
It's a sample from OpenGameArt and listed under pulic domain. I can try to find a shorter one.